frontend/pages/Home.js

```diff
@@ -11,11 +12,22 @@ export class HomePage extends React.Component {
 
     render() {
         return (
+            // <StrictMode>
+            //     <Collage/>
```

Comment:
It's generally good practice to remove dead code before submitting a PR. You can also remove the import  for `StrictMode` on line 1, as it's no longer needed.